scopes/lanes/ui/navigation/lane-switcher/lane-switcher.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { HTMLAttributes } from 'react';
```

Comment:
can be removed from this PR 
Will be fixed in #6714 

https://github.com/teambit/bit/pull/6714/files#diff-e764c0dcda6133bbea98ddc9602826023931e423a209ad2bc3c21c87af3be9ab